src/segmented-control/segmented-control.scss

```diff
@@ -13,6 +13,11 @@
   li {
     position: relative;
     display: inline-flex;
+
+    // stylelint-disable-next-line selector-max-compound-selectors
+    &:has([aria-current='true']) + li .SegmentedControl-button::before {
```

Comment:
`:has` doesn't have good browser support: https://caniuse.com/css-has

Reply:
Can't really think of a way to replace `has()`, except maybe moving the `aria-current='true'` to the parent `li` or some negative margin hack.

Just curious about comment https://github.com/primer/design/pull/276#issuecomment-1205336999

> a segmented control's keyboard navigation should be like a list (use tabs to focus buttons)

@mperrotti did you mean to literally use a `<ul>` and wrap the buttons in `<li>`s, or did you mean that the focus should just move from one button to the next (like in a list) when pressing `tab`. If the later, could we also just change the `role="toolbar"` to `role="list"` instead?

Reply:
I changed it to semantically be a `<ul>` in Primer React. Would you propose different markup?

Reply:
I tried at first relying on role only, but axe didn't like `role="list" > button` buttons being direct descendants of role list, and I couldn't apply role=listitem to the button

Reply:
Yea, buttons should stay `role="button"`, and we shouldn't have descendants of `role="list"` that aren't `role="listitem"`